public/css/genericStyles.css

```diff
@@ -0,0 +1,7 @@
+* {
```

Comment:
why not just use bootstrap? no css then.

Reply:
group consensus.